assets/src/edit-story/components/panels/animation/effectInput.js

```diff
@@ -51,18 +55,42 @@ function EffectInput({
   effectConfig,
   field,
   onChange,
+  onMultiChange,
   disabledOptions,
 }) {
   const rangeId = `range-${uuidv4()}`;
   const directionControlOnChange = useCallback(
-    ({ nativeEvent: { target } }) => onChange(target.value, true),
-    [onChange]
+    ({ nativeEvent: { target } }) => {
+      if (
+        target.value === SCALE_DIRECTION.SCALE_IN ||
+        target.value === SCALE_DIRECTION.SCALE_OUT
+      ) {
+        onMultiChange(
```

Comment:
We use a multi change function because changing scale in or out actually changes two values behind the scenes. This is the only case where a single click in the direction picker changes two values and they must happen at the same time. There isn't a great way to manage this other than an if-statement since there isn't a unified formula here.

Reply:
I think updating the effect here to take a direction instead of a from & to value could remove the need to do this